conf/cassandra.yaml

```diff
@@ -1375,6 +1375,21 @@ dynamic_snitch_reset_interval: 600000ms
 # until the pinned host was 20% worse than the fastest.
 dynamic_snitch_badness_threshold: 1.0
 
+# Configures Java crypto provider. By default, it will use DefaultCryptoProvider
+# which will install Amazon Correto Crypto Provider.
+#
+# Amazon Correto Crypto Provider works currently for x86_64 and aarch_64 platforms.
```

Comment:
Good idea but what if that project changes the name or something like that ... links tend to break.

Reply:
This is up to you :-)
I think if the link can save a user a few seconds, it's better to add it. In case it gets broken at some point in the future then we have an LHF for someone to fix.

Reply:
I wont add it there. The link would be something for "ah it is this thing" rather than a link which is crucial for the configuration itself. There are plethora of cases where links like this might be added over all documentation. Are we going to add links to Facebook's Zstd so a user knows what Zstd is? etc etc